tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_hdinsight_26838.go

```diff
@@ -0,0 +1,78 @@
+package dataworkarounds
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
+
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
```

Comment:
```suggestion
	"fmt"
	
	"github.com/hashicorp/go-azure-helpers/lang/pointer"
	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
```

Reply:
huh - I could have sworn I'd pushed that 😬 - at any rate thanks, will apply that 👍